src/services-managing.rst

```diff
@@ -278,13 +241,16 @@ Managing Tables Settings
 
 Managing Device Settings
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+:menuselection:`Settings --> Prefrences`
 
   .. image:: /img/services-managing/services-admin-device.*
     :alt: Services Admin Device Settings
     :class: device-screen-vertical
 
+  - :menuselection:`Default Locale` - specifies your preferred localization. By default this is set to US English. If you provide translations for your Data Management Application, this is where to enable them.
```

Comment:
"If you provide translations for your Data Management Application" - not clear what Data Management Application means here.

Reply:
@Goodiec comment still pending.

Reply:
This refers to the application the user is using for data management (collecting, storing, and processing data), in this case the ODK-X tools.